### PR TITLE
[5] Investigate how to remove price range filter and remove it

### PR DESCRIPTION
**Pull request by iggysav**
https://trello.com/c/1B1RWflA/5-investigate-how-to-remove-price-range-filter-and-remove-it

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -74,7 +74,8 @@ end
 gem 'solidus', '~> 4.2', '>= 4.2.3'
 
 
-gem "solidus_auth_devise", "~> 2.5"
+ gem "solidus_auth_devise", "~> 2.5"
```
Comment (fooooxmr) [inline Gemfile]: Why we have changes here?

File: Gemfile.lock [hunk 2]
```diff
@@ -130,10 +130,10 @@ GEM
       activerecord (>= 4.2, < 8)
     erubi (1.12.0)
     execjs (2.9.1)
-    factory_bot (6.4.0)
+    factory_bot (6.2.1)
```
Comment (fooooxmr) [inline Gemfile.lock]: And here?

**Review by fooooxmr — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (iggysav) [reply Gemfile]: Probably an extra space. This was left over from the first attempt to solve the issue by disabling gem authorization. But then the login for admin stopped working. So I uncommented this gem, but the space was left)).

File: Gemfile.lock (hunk 2 above)
Comment (iggysav) [reply Gemfile.lock]: Hmm. I'm having trouble answering that one. Pulled it up from some experimentation. Remove Gemfile.lock from this pull request.

Comment (fooooxmr) [reply Gemfile.lock]: Please remove changes from Gemfile and Gemfile.lock in this PR.

**Review by fooooxmr — APPROVED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (iggysav) [reply Gemfile.lock]: Gemfile and Gemfile.lock have been removed

**Review by fooooxmr — APPROVED** [review]

**Pull request merged**